basicdataset/src/test/resources/mlrepo/dataset/tabular/ai/djl/basicdataset/daily-delhi-climate/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "metadataVersion": "0.2",
+  "resourceType": "dataset",
+  "application": "tabular",
+  "groupId": "ai.djl.basicdataset",
+  "artifactId": "daily-delhi-climate",
+  "name": "daily-delhi-climate",
+  "description": "This dataset provides data from 1st January 2013 to 24th April 2017 in the city of Delhi, India. The 4 parameters here are meantemp, humidity, wind_speed, meanpressure.",
+  "website": "https://www.kaggle.com/datasets/sumanthvrao/daily-climate-time-series-data",
+  "licenses": {
+    "license": {
+      "name": "CC0 1.0 Universal",
+      "url": "https://creativecommons.org/publicdomain/zero/1.0/legalcode"
+    }
+  },
+  "artifacts": [
+    {
+      "version": "3.0",
```

Comment:
Any reason this version is "3.0"?

Reply:
On the Kaggle page, the author marks the latest version of this dataset `Version 3`, so I just wrote "3.0" here. Of course we can change it into "1.0" if you think it's better.